Gopkg.toml

```diff
@@ -0,0 +1,23 @@
+[[constraint]]
+  branch = "master"
+  name = "github.com/container-storage-interface/spec"
+
+[[constraint]]
+  branch = "master"
+  name = "github.com/golang/glog"
+
+[[constraint]]
+  name = "google.golang.org/grpc"
+  version = "1.7.2"
+
+[[constraint]]
```

Comment:
why do we need this?

Reply:
The version constraint was taken from kubernetes-csi, and grpc bits are used in nodeserver.go
pkg/rbd/nodeserver.go:  "google.golang.org/grpc/codes"
pkg/rbd/nodeserver.go:  "google.golang.org/grpc/status"